src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import 'blocks/card';
+@import 'blocks/stars';
+
 body {
   margin: 0;
+  padding: 50px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use zero values instead of `0px`. So you should use `0` instead of `50px` for padding if that meets the design requirements.